src/tests/vos_engine.c

```diff
@@ -76,8 +76,6 @@ engine_cont_init(struct credit_context *tsc)
 	if (rc)
 		return rc;
 
-	vos_pool_features_set(tsc->tsc_poh, VOS_POOL_FEAT_AGG_OPT);
-
```

Comment:
By reading the changes of current patch, I don't see why the "features_set" call for standalone VOS could be removed now, could you elaborate it?

Reply:
In pool open, we do this

772         if (pool_df->pd_version >= POOL_DF_AGG_OPT)
 773                 pool->vp_feats |= VOS_POOL_FEAT_AGG_OPT;

The reason we had to set the feats manually was due to pd_version still being old version despite upgrading the pool.

Reply:
oh, this means we could not downgrade daos again? i think original plan is DAOS could be downgraded to lower version if this pool has not been run by 'dmg pool upgrade', it looks this PR break this?